test/k8sT/Chaos.go

```diff
@@ -24,7 +24,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("K8sChaosTest", func() {
+var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sChaosTest", func() {
```

Comment:
Maybe add a comment to tell why we skip on 5.4? (Here and in other files.) This would avoid searching through git history to understand the reason?